hugo/content/species/aspergillus_nidulans/_index.md

```diff
@@ -12,6 +12,7 @@ img_attrib_text: "[EDIT]"
 img_attrib_link: [EDIT]
 
 lineage_data_path: "aspergillus_nidulans/taxonomy"
+jbrowse_config: "/data/aspergillus_nidulans/config.json"
```

Comment:
If we are consistent (and who wouldn't strive for that), `species/<latin name>/_index.md` references `/data/<latin name>/config.json` so `jbrowse_config` could be inferred. This would also avoid typos. I don't know the right way to achieve this though. Maybe using Hugo [archetypes](https://gohugo.io/content-management/archetypes/)  for species, and the [`Dir`](https://gohugo.io/methods/page/file/#dir) method ? That would be fancy, we could do `hugo new content species/<latin name>/_index.md` 😎

Reply:
Great idea! I have implemented it now with Hugo in commit [79c3ce3](https://github.com/ScilifelabDataCentre/swedgene/pull/11/commits/79c3ce38d31bda8ab8fb6c7983be4644bb4441ab) 

I have made a note to also do this with some other params, but will save that for another time as a bit out of scope of this PR.